src/Resources/views/CRUD/Association/edit_one_to_many_inline_tabs.html.twig

```diff
@@ -20,7 +20,7 @@ file that was distributed with this source code.
                                 data-toggle="tab"
                             >
                                 <i class="icon-exclamation-sign has-errors hide"></i>
-                                {{ name|trans({}, form_group.translation_domain) }}
+                                {{ form_group.label|trans({}, form_group.translation_domain ?: associationAdmin.translationDomain) }}
```

Comment:
I guess both variants behave the same way, this comment is just to keep consistency.
```suggestion
                                {{ form_group.label|trans({}, form_group.translation_domain|default(associationAdmin.translationDomain)) }}
```

Reply:
Sure, done !